functions.php

```diff
@@ -73,65 +114,75 @@ public function register_taxonomies() {
 	 * @param  wp_query.
 	 */
 	public function tags_support_query( $wp_query ) {
-		if ( $wp_query->get('tag') ) {
+		if ( $wp_query->get( 'tag' ) ) {
 			$wp_query->set( 'post_type', 'any' );
 		}
 
-		if ( $wp_query->get('category_name') ) {
+		if ( $wp_query->get( 'category_name' ) ) {
 			$wp_query->set( 'post_type', 'any' );
 		}
 	}
 
-	/**
-	 * Hook in and add a Theme metabox. Can only happen on the 'cmb2_admin_init' or 'cmb2_init' hook.
-	 */
+	//Hook in and add a Theme metabox. Can only happen on the 'cmb2_admin_init' or 'cmb2_init' hook.
 	public function register_header_metabox() {
 
 		$prefix = 'p4_';
 
-		$p4_header = new_cmb2_box( array(
-			'id'            => $prefix . 'metabox',
-			'title'         => esc_html__( 'Page Header Fields', 'planet4-master-theme' ),
-			'object_types'  => array( 'page' ), // Post type
-		) );
-
-		$p4_header->add_field( array(
-			'name' => esc_html__( 'Header Title', 'planet4-master-theme' ),
-			'desc' => esc_html__( 'Header title comes here', 'planet4-master-theme' ),
-			'id'   => $prefix . 'title',
-			'type' => 'text_medium',
-		) );
-
-		$p4_header->add_field( array(
-			'name' => esc_html__( 'Header Subtitle', 'planet4-master-theme' ),
-			'desc' => esc_html__( 'Header subtitle comes here', 'planet4-master-theme' ),
-			'id'   => $prefix . 'subtitle',
-			'type' => 'text_medium',
-		) );
-
-		$p4_header->add_field( array(
-			'name'    => esc_html__( 'Header Description', 'planet4-master-theme' ),
-			'desc'    => esc_html__( 'Header description comes here', 'planet4-master-theme' ),
-			'id'      => $prefix . 'description',
-			'type'    => 'wysiwyg',
-			'options' => array(
-				'textarea_rows' => 5,
-			),
-		) );
-
-		$p4_header->add_field( array(
-			'name' => esc_html__( 'Header Button Title', 'planet4-master-theme' ),
-			'desc' => esc_html__( 'Header button title comes here', 'planet4-master-theme' ),
-			'id'   => $prefix . 'button_title',
-			'type' => 'text_medium',
-		) );
-
-		$p4_header->add_field( array(
-			'name' => esc_html__( 'Header Button Link', 'planet4-master-theme' ),
-			'desc' => esc_html__( 'Header button link comes here', 'planet4-master-theme' ),
-			'id'   => $prefix . 'button_link',
-			'type' => 'text_medium',
-		) );
+		$p4_header = new_cmb2_box(
+			array(
```

Comment:
Actually previous syntax is correct, where `array(` appears on the same line after the method name, i.e.

This is correct:
```
new_cmb2_box( array(
```

This is incorrect:
```
new_cmb2_box(
    array(
```

Reply:
Also, please fix coding standards (spacing, ect) only in the code that's directly being edited/altered or added. If you see some coding standards issues in the code that's not being directly edited and want to fix that - it's really great! but please create a separate PR just for that. Otherwise, it's hard to track what's changed and github isn't as smart highlighting diffs. Thanks :)

Reply:
Yes agreed. I will keep this in mind :)